integration-tests/gaudi/test_gaudi_generate.py

```diff
@@ -25,6 +31,7 @@
             "--max-batch-prefill-tokens",
             "2048",
         ],
+        "run_by_default": True,
```

Comment:
I cannot add the comment at line 24 so adding it here. Related to my other comment, let's test this model on two devices only (so 2 shards). This allows us to use a smaller runner and should be a bit faster.